sp800-63-3/sec4_model.md

```diff
@@ -8,14 +8,11 @@
 
 ### <a name="4-1"></a>4.1. Overview 
 
-
-Digital identity is the unique representation of a subject engaged in an online transaction. Identity proofing is a process used to verify an applicant's association with their real world identity. Digital authentication is the process of establishing confidence in individual identities presented digitally to a system. Systems can use the authenticated identity to determine if a subject is authorized to perform an online transaction. In most cases, the authentication and transaction take place across an open network such as the Internet; however, in some cases, access to the network may be limited and authorization decisions may take this into account.
-
 The digital identity model used in these guidelines reflects current technologies and architectures available in the market. More complex models that separate functions - such as issuing credentials and providing attributes - among larger numbers of parties are also available and may have advantages in some classes of applications. While a simpler model is used in this document, it does not preclude agencies from separating these functions. In addition, certain enrollment, identity proofing, and issuance processes performed by the CSP are sometimes delegated to an entity known as the either RA or the identity manager (IM). A close relationship between the RA/IM and CSP is typical, and the nature of this relationship may differ among RAs, IMs, and CSPs. The types of relationship and their requirements is outside of the scope of this document.  Accordingly, the term CSP will be used to be inclusive of RA and IM functions. Finally, a CSP may provide other services in addition to digital identity services. In these situations, the requirements specified throughout these documents only apply to the CSP function(s), not the additional services.
 
-Digital authentication begins with enrollment. The usual sequence for enrollment proceeds as follows. An applicant applies to a CSP. If approved, the CSP creates a credential and binds it to one or more authenticators. The credential includes at least one identifier, which can be pseudonymous, and possibly one or more attributes that the CSP has verified. The authenticators may be issued by the CSP, provided directly by the subscriber, or provided by a third party. The authenticators and credential may be used in subsequent authentication events.
```

Comment:
There is now no mention of the binding of authenticators to identity as part of the enrollment process. Is that an oversight?